### ISSUE TITLE
How to keep nav collapsed / expanded with its original status when back?

### ISSUE THREAD

**Issue by aloveric** [andreamazz/AMScrollingNavbar#265]
![img_0827](https://user-images.githubusercontent.com/8198881/29406495-d579c8f8-8373-11e7-908a-d14e36cb588e.JPG)

**Comment by andreamazz** [comment]
Hi @aloveric 
I'm not sure I fully understand, can you elaborate a bit more?

**Comment by andreamazz** [comment]
Hey @pablogeek 
I've just tested this, and it seems to be working fine. I've pushed a new controller when the navbar is hidden, once I got back it was still collapsed.

**Comment by pablogeek** [comment]
@andreamazz This happens when you use `present` instead of `pushViewController`

```
let storyBoard = UIStoryboard(name: "Main", bundle: nil)
    let mainViewController = storyBoard.instantiateViewController(withIdentifier: "ViewController")
    self.present(mainViewController, animated: true, completion: nil)
```

If I do this, when I dismiss the `mainViewController` it will show the navbar expanded

**Comment by pablogeek** [comment]
@andreamazz any news on this? thanks!

**Comment by andreamazz** [comment]
Yeah, UIKit restores the navbar to its full height after a modal presentation, the only thing I can do for now is restore the alpha of its subviews. I can't get it to stay hidden. Any help is welcome.

**Comment by andreamazz** [comment]
Screw it, I'll have to rewrite part of the state handling, it's well overdue. I'll kee you posted.
